test/util_prop.ml

```diff
@@ -0,0 +1,20 @@
+open Generator.Test
+open Pollinate.Util
+open Commons
+
+let pack_unpack =
+  QCheck2.Test.make 
+  ~count:1000
+  ~name:"unpack . pack returns the original value"
+  request_gen
+  (fun random_request -> 
+    let open Commons in 
+    random_request == 
```

Comment:
In OCaml, we typically want `=` for equality checking, not `==`. https://stackoverflow.com/questions/13590307/whats-the-difference-between-equal-and-identical-in-ocaml